### PR TITLE
UseScrollSpy cleanup

### PR DESCRIPTION
**Pull request by kristian240**
* move `jest` to root
* add useScrollSpy README

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -16,6 +16,6 @@
 		"allowSyntheticDefaultImports": true,
 		"downlevelIteration": true
 	},
-	"include": ["@types", "packages"],
+	"include": ["@types", "packages", "jest.config.js"],
```
Comment (isBatak) [inline tsconfig.json:19]: It's `.js` file

Comment (kristian240) [reply tsconfig.json:19]: It's automatically added.

Comment (isBatak) [reply tsconfig.json:19]: by which tool?

Comment (kristian240) [reply tsconfig.json:19]: TS

**Pull request merged**